Makefile

```diff
@@ -9,13 +11,20 @@ codespell.1: codespell.1.include bin/codespell
 	sed -i '/\.SS \"Usage/,+2d' codespell.1
 
 check-dictionary:
-	@if ! LC_ALL=C sort ${SORT_ARGS} -c codespell_lib/data/dictionary.txt; then \
+	@if ! LC_ALL=C sort ${SORT_ARGS} -c ${DICTIONARY}; then \
 		echo "Dictionary not sorted. Sort with 'make sort-dictionary'"; \
 		exit 1; \
 	fi
+	@if egrep -n "^\s*$$|\s$$|^\s" ${DICTIONARY}; then \
+		echo "Dictionary contains leading/trailing whitespace and/or blank lines.  Trim with 'make trim-dictionary'"; \
+		exit 1; \
+	fi
 
 sort-dictionary:
-	LC_ALL=C sort ${SORT_ARGS} -u -o codespell_lib/data/dictionary.txt codespell_lib/data/dictionary.txt
+	LC_ALL=C sort ${SORT_ARGS} -u -o ${DICTIONARY} ${DICTIONARY}
+
+trim-dictionary:
+	sed -E -i.bak -e 's/^[[:space:]]+//; s/[[:space:]]+$$//; /^$$/d' ${DICTIONARY} && rm ${DICTIONARY}.bak
```

Comment:
I don't know about you, but I regularly use .bak as a backup extension when hacking around with stuff, won't sed overwrite that file, and the rm remove it. Something more unique, like adding the PID would be safer @warmsocks .

Reply:
I'll submit a PR that uses ex instead of sed, so no tempfile is required at all.